src/svg.rs

```diff
@@ -130,9 +156,16 @@ mod test {
     }
 
     #[test]
-    #[cfg(unix)]
+    fn without_file() {
+        let surface = SvgSurface::new(100., 100., None::<&Path>).unwrap();
+        draw(&surface);
+        surface.finish();
+    }
+
+    #[test]
     fn file() {
-        let surface = SvgSurface::new(100., 100., "/dev/null").unwrap();
+        let file = NamedTempFile::new().expect("tempfile failed");
```

Comment:
Why not `tempfile()` ?

Reply:
Because I couldn't find a way to get a `Path` from `tempfile()`

Reply:
Your right